azkaban-common/src/main/java/azkaban/jobtype/JobTypeManager.java

```diff
@@ -124,7 +124,7 @@ private void loadPluginJobTypes(final JobTypePluginSet plugins)
     } else {
       logger.info("Common plugin job props file " + commonJobPropsFile
           + " not found. Using empty props.");
```

Comment:
Actually, one last thing. Can you have this comment say `Using only globals props.` instead of `Using empty props.`?

That will make this logger message line up with the new logic. Thanks! :)